java/code/src/com/suse/manager/reactor/utils/RhelUtils.java

```diff
@@ -160,15 +162,24 @@ public String getRelease() {
             String minorVersion = StringUtils.substringAfter(matcher.group(2), ".");
             String release = matcher.group(3);
             return Optional.of(new ReleaseFile(name, majorVersion, minorVersion, release));
-        }
-        else {
-            Matcher omatcher = ORACLE_RELEASE_MATCHER.matcher(releaseFile);
-            if (omatcher.matches()) {
+        } else { // TODO ALIBABA VERIFY THIS IS CORRECT
```

Comment:
Was this checked? Please remove the TODO

Reply:
Removed